src/App.jsx

```diff
@@ -14,45 +17,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALPHAB = 'name';
+const SORT_BY_LENGTH = 'length';
+
+function getSortedGoods(goodsArr, sortField) {
+  if (sortField !== '') {
+    goodsArr.sort((good1, good2) => {
```

Comment:
let's use 'a' and 'b' instead of good1 and good2, using numbers in variable names is not the best practice, everyone knows that 'a' and 'b' are goods